server/services/teamsSummaryCountService.test.ts

```diff
@@ -0,0 +1,471 @@
+// Mock dependencies
+// Import the class under test
+import TeamsSummaryCountService from './teamsSummaryCountService'
+
+// Mock the problematic imports
+jest.mock('../../logger', () => ({
+  info: jest.fn(),
+  error: jest.fn(),
+}))
+
+const mockGetTeam = jest.fn()
+const mockGetProduct = jest.fn()
+const mockLogger = jest.requireMock('../../logger')
+
+jest.mock('../data', () => ({
+  StrapiApiClient: jest.fn().mockImplementation(() => ({
+    getTeam: mockGetTeam,
+    getProduct: mockGetProduct,
+  })),
+}))
```

Comment:
better to use `jest.mock`

good example in `ComponentNameService.test.ts`:

```ts
jest.mock('../data/strapiApiClient')

const strapiApiClient = new StrapiApiClient() as jest.Mocked<StrapiApiClient>
```
this will let you just do this: 
```ts
strapiApiClient.getTeam.mockedResolve(...
```

```